### PR TITLE
FBC-259 and LOAN-158 - Eliminate circular owl:imports

### PR DESCRIPTION
**Pull request by ElisaKendall**
Signed-off-by: Elisa Kendall <ekendall@thematix.com>

## Description

Eliminate circular imports in FBC Business Centers and LOAN HMDA, and revise definitions in Business Centers to eliminate circularity and make them ISO 704 compliant

Fixes: #1152 / FBC-259, LOAN-158

## Checklist:

- [x] I'm familiar with the [FIBO developer quide](../CONTRIBUTING.md#contributing-to-the-fibo-code). My contribution meets all the requirements described there.
- [x] My contribution follows the [principles of best practices for FIBO](../ONTOLOGY_GUIDE.md).
- [x] My changes have been reconciled with latest master and no merge conflicts remain.
- [x] This PR is related to exactly one issue. The issue is referenced by using a GitHub keyword such as "fixes", "closes", or "resolves".
- [x] Hygiene tests have been applied by a PR with "(WIP)" in title.
- [x] The issue has been tested locally using a reasoner (for ontology changes).

### CODE REVIEW

**Review by dallemang — APPROVED** [review]
How did this go through?  Don't we have a hygiene test for this?   If not, we should. 

@mereolog

**Review by trypuz — APPROVED** [review]
Approved

File: FBC/FunctionalEntities/BusinessCenters.rdf [hunk 1]
```diff
@@ -66,14 +66,15 @@
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label>business center code</rdfs:label>
-		<skos:definition>a code used to denote a municipality or business center</skos:definition>
+		<skos:definition>code used to denote a metropolitan area where business is conducted</skos:definition>
```
Comment (rivettp) [inline FBC/FunctionalEntities/BusinessCenters.rdf:69]: Not sure this captures the notion of "business center" as used in FIBO where there is more of a emphasis on financial business activity; 
And the Code is in the FpML Scheme (should we not have an individual of class Scheme?)

File: FBC/FunctionalEntities/BusinessCenters.rdf [hunk 2]
```diff
@@ -92,7 +93,7 @@
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label>business day adjustment code</rdfs:label>
-		<skos:definition>a code used to denote convention for specifying what happens when a date falls on a day that is weekend or holiday in some municipality or business center</skos:definition>
+		<skos:definition>code used to denote a convention for specifying what happens when a date falls on a day that is weekend or holiday in some municipality or business center</skos:definition>
```
Comment (rivettp) [inline FBC/FunctionalEntities/BusinessCenters.rdf:96]: should not say "some municipality" but "the municipality relevant to the contract or transaction"

**Review by rivettp — APPROVED** [review]
Some comments that are not showstoppers but should be addressed - either in this PR or via separate issues

**Pull request merged**